### PR TITLE
add batchnorm3d module

### PR DESCRIPTION
**Pull request by YongtaoShi**
image

### CODE REVIEW

File: python/oneflow/test_utils/automated_test_util/torch_flow_dual_object.py [hunk 1]
```diff
@@ -175,9 +177,11 @@ def __init__(self, name, pytorch, oneflow):
             state_dict = pytorch.state_dict()
             state_dict = {k: v.detach().cpu().numpy() for (k, v) in state_dict.items()}
             oneflow.load_state_dict(state_dict)
-            dual_modules_to_test.append(self)
+            if testing:
+                dual_modules_to_test.append(self)
         if isinstance(pytorch, torch_original.Tensor):
-            dual_objects_to_test.append(self)
+            if testing:
+                dual_objects_to_test.append(self)
```
Comment (daquexian) [inline python/oneflow/test_utils/automated_test_util/torch_flow_dual_object.py:184]: NOTE: 这个文件是我改的

**Review by Flowingsun007 — APPROVED** [review]
